db/config.cc

```diff
@@ -362,6 +362,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "If set to higher than 0, ignore the controller's output and set the compaction shares statically. Do not set this unless you know what you are doing and suspect a problem in the controller. This option will be retired when the controller reaches more maturity")
     , compaction_enforce_min_threshold(this, "compaction_enforce_min_threshold", liveness::LiveUpdate, value_status::Used, false,
         "If set to true, enforce the min_threshold option for compactions strictly. If false (default), Scylla may decide to compact even if below min_threshold")
+    , compaction_twcs_offstrategy_enabled(this, "compaction_twcs_offstrategy_enabled", liveness::LiveUpdate, value_status::Used, true,
```

Comment:
I know today we only have issues with twcs, but I hate the fact we're introducing a hard-coded twcs-specific mechanism.
Let's consider an array of compaction strategy names here so it could be used to control offstrategy for any compaction strategy.

Reply:
Otherwise, LGTM

Reply:
If you want it per strategy, maybe it makes sense to make it a compaction property inherited by all strategies, so we can change it on schema level. It's usually a single TWCS table affected anyway. @vladzcloudius does that work for you?